TownSuite.Web.ImageGen.Tests/GenerateIdenticonImageRepoTest.cs

```diff
@@ -14,16 +12,14 @@ public void Setup()
     {
     }
 
-    private static string[] ImageFormatCases = new string[] { "jpeg", "png", "gif", "webp" };
-    
-    [Test, TestCaseSource("ImageFormatCases")]
+    private static string[] ImageFormatCases = new string[] { "jpeg", "png", "gif", "webp", "avif", "heic" };
+
+    [Test, TestCaseSource(nameof(ImageFormatCases))]
```

Comment:
We use [the nameof operator](https://docs.microsoft.com/dotnet/csharp/language-reference/operators/nameof) to avoid introducing [magic strings](https://wikipedia.org/wiki/Magic_string) into code, which offers better resilience when refactoring. While nameof is recommended, you could also use the string "DivideCases" to achieve the same outcome.